js/private/node_package.bzl

```diff
@@ -27,6 +26,13 @@ _ATTRS = {
 Can be left unspecified to allow for circular deps between `node_package`s.        
 """,
     ),
+    "bins": attr.string_dict(
+        doc = """A dict of bin names to paths for this package""",
```

Comment:
could you be more clear what is a "bin name"

Reply:
✅